spidermon/contrib/scrapy/monitors.py

```diff
@@ -87,23 +88,60 @@ def test_should_finish_with_expected_reason(self):
 class UnwantedHTTPCodesMonitor(BaseScrapyMonitor):
     """Check for maximum number of unwanted HTTP codes.
 
-    You can configure a ``dict`` of unwanted HTTP codes with
+    Usage # 1
+
+    You can configure an ``int`` of maximum unwanted HTTP codes with
+    ``SPIDERMON_UNWANTED_HTTP_CODES_MAX_COUNT`` the default value is::
+
+        SPIDERMON_UNWANTED_HTTP_CODES_MAX_COUNT = 10
```

Comment:
It is not clear where the user needs to change this values, neither what the section after the `----->` means.

Reply:
part after `---->`
Shows where the value mentioned will be used.

Reply:
-----> removed.